restart.sh

```diff
@@ -1,2 +1,2 @@
-kill `ps -ae | grep python | cut -d ' ' -f 1`
+kill "$(pgrep -a python | awk '{$1=$1};1' | cut -d ' ' -f 1)"
```

Comment:
why pgrep + awk instead of ps + grep?

Reply:
awk is used to make the command platform independent: while Macs put one space at the start of ps, Linux puts 3 and Windows puts 4

pgrep was suggested by Max's Pycharm as an alternative to ps + grep